src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="/search"
+       method="GET"
+       data-qa="big"
+       class="search-bar">
```

Comment:
Checklist point number 5.
```suggestion
    <form
       action="/search"
       method="GET"
       data-qa="big"
       class="search-bar"
    >
```